apps/coa/translations/src/en/pages.json

```diff
@@ -1,3 +1,54 @@
 {
-  
+    "overview": {
+        "header": "Update your details without an account"
+    },
+    "applicant-details": {
+        "header": "Applicant details",
+        "application-details-intro": "Enter the details of the person whose details you want to update. Their details must match the identity document that UK Visas and Immigration holds on record."
+    },
+    "who": {
+        
+    },
```

Comment:
You've got an empty configuration for the "who" page. Might be better to set the "header" here, and in the /fields/index.js file, you can set the `isPageHeading` to false for the `who-are-you` field

Reply:
As this is one question per page, `legend` is used as the page heading for this Radios component,  as per [GDS recommendation](https://design-system.service.gov.uk/components/radios/#:~:text=If%20you%E2%80%99re%20asking,the%20contents%20once.)